docs/ss_slide_007.md

```diff
@@ -7,5 +7,5 @@
 
 
 {:style="text-align:center;"}
-[Back to first slide](https://rivesunder.github.io/yuca/ss_slide_000) -- [Previous slide](https://rivesunder.github.io/yuca/ss_slide_006) -- [Some CCA references](https://rivesunder.github.io/yuca/g_ref) -- [Supporting resources index](https://rivesunder.github.io/yuca/ss_slide_000)
+[Back to first slide](https://rivesunder.github.io/yuca_docs/ss_slide_000) -- [Previous slide](https://rivesunder.github.io/yuca_docs/ss_slide_006) -- [Some CCA references](https://rivesunder.github.io/yuca_docs/g_ref) -- [Supporting resources index](https://rivesunder.github.io/yuca_docs/ss_slide_000)
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.